examples/kubernetes/kuma-demo/kuma-demo-local.yaml

```diff
@@ -0,0 +1,188 @@
+apiVersion: v1
```

Comment:
Do we still want to keep aio.yaml?

Reply:
yaroslav is making changes for it to be used in the workshop environment, that is why im separating it out